tedana/decomposition/eigendecomp.py

```diff
@@ -202,11 +202,9 @@ def tedpca(catd, OCcatd, combmode, mask, t2s, t2sG, stabilize,
         vTmix = comp_ts.T
         vTmixN = ((vTmix.T - vTmix.T.mean(0)) / vTmix.T.std(0)).T
         LGR.info('Making initial component selection guess from PCA results')
-        _, ct_df, betasv, v_T = model.fitmodels_direct(catd, comp_ts.T, eimum,
-                                                       t2s, t2sG,
-                                                       tes, combmode, ref_img,
-                                                       mmixN=vTmixN,
-                                                       full_sel=False)
+        _, ct_df, betasv, v_T = model.fitmodels_direct(
+            catd, comp_ts.T, eimum, t2s, t2sG, tes, combmode,
+            ref_img, mmixN=vTmixN, full_sel=False, label='pca')
```

Comment:
I'm not sure that we want to add an additional label argument here, since we already have a label argument one level up !

Reply:
`fitmodels_direct` is a general function that's run for both PCA and ICA, so any files that are generated need to be identified as either PCA- or ICA-related. I'll move it off to another PR though.

Reply:
No no, sorry, I understand the point ! I do think we should move it to another PR for the binder-related reasons, but I just mean the argument itself could be called something other than "label". Otherwise we have two, unrelated "label" arguments floating around for writing out files !

Reply:
Just reverted the walkthrough-related changes, so we can revisit the setup for `fitmodels_direct` in that PR.